vanilla_js/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Pelm Connect Javascript Demo</title>
+        <script src="https://cdn.pelm.com/initialize.js"></script>
+        <script defer>
+            (async () => {
+                // Create a connect token using client credentials in order to initiate the Connect Flow.
+                const generateConnectToken = () => {
+                    return fetch('/connect')
+                        .then((r) => r.json())
+                        .then((d) => {
+                            return d.connectToken;
+                        });
+                };
+
+                // Sending request to our backned at the accounts endpoint
+                const generateAccounts = async () => {
+
+                    const accounts = await fetch('/accounts')
+                        .then((r) => r.json())
+                        .then((data) => JSON.stringify(data, null, 2));
+
+                    const gettingAccounts = document.getElementById('getting-accounts');
```

Comment:
could you post a screenshot of what this page looks like?

Reply:
For sure, sending to you now